pyproject.toml

```diff
@@ -49,7 +49,7 @@ docs = [
     "imgui-bundle>=1.2.1",
 ]
 tests = ["pytest", "psutil", "trimesh", "httpx", "gltflib", "imageio"]
-dev = ["pygfx[lint,tests,examples,docs]"]
+dev = ["pygfx[lint,tests,examples,docs]", "open3d"]
```

Comment:
Can you please add it to the list of example deps instead? (The `dev` deps include the `examples` deps).

Reply:
But won't it fail then on 3.12?

Reply:
Oh right ... 🤔  Perhaps its better to not specify it as a dependency. If someone runs the example, that person will have to manually install open3d for the time being.

Reply:
I've added the same `find_spec` to the open3d examples for now and removed the dependency from pygfx completely.